### PR TITLE
fix:reset default auto index type for scalar

### PR DESCRIPTION
**Pull request by zhagnlu**
#32900

### CODE REVIEW

File: pkg/util/paramtable/autoindex_param.go [hunk 1]
```diff
@@ -170,7 +170,7 @@ func (p *autoIndexConfig) init(base *BaseTable) {
 	p.ScalarAutoIndexParams = ParamItem{
 		Key:          "scalarAutoIndex.params.build",
 		Version:      "2.4.0",
-		DefaultValue: `{"int": "HYBRID","varchar": "HYBRID","bool": "BITMAP", "float": "INVERTED"}`,
+		DefaultValue: `{numeric": "INVERTED","varchar": "INVERTED","bool": "INVERTED"}`,
```
Comment (czs007) [inline pkg/util/paramtable/autoindex_param.go:173]: better to keep bool as BITMAP

Comment (zhagnlu) [reply pkg/util/paramtable/autoindex_param.go:173]: keep all same as 2.4 is better? May we need user to set it by themself which is better for compatibility，they know what changed.

**Pull request merged**